### PR TITLE
Backport legion hid2 to 1.9.x

### PR DESCRIPTION
**Pull request by superm1**

### CODE REVIEW

File: plugins/legion-hid2/fu-legion-hid2.rs [hunk 1]
```diff
@@ -3,65 +3,65 @@
...
+enum LegionHid2Command {
     GetVersion = 0x01,
     GetMcuId = 0x02,
     StartIap = 0xE1,
     IcReset = 0xEF,
 }
 
 #[repr(u8)]
-enum FuLegionHid2ReportId {
+enum LegionHid2ReportId {
```
Comment (hughsie) [inline plugins/legion-hid2/fu-legion-hid2.rs:14]: You don't need to do this since d69307979affd77624c31dd932370834b57b3c08

Comment (superm1) [reply plugins/legion-hid2/fu-legion-hid2.rs:14]: In 1.9.x?

Comment (hughsie) [reply plugins/legion-hid2/fu-legion-hid2.rs:14]: Yes, it's how I did the jabra-file backport.

Comment (superm1) [reply plugins/legion-hid2/fu-legion-hid2.rs]: Ah I see, thanks!  Fixed.

**Pull request merged**